pyaedt/generic/general_methods.py

```diff
@@ -1062,6 +1062,7 @@ def __init__(self):
         self.formatter = None
         self.remote_rpc_session = None
         self.remote_rpc_session_temp_folder = ""
+        self.remote_rpc_service_manager_port = 17878
```

Comment:
@maxcapodi78 Was it working previously without hard-coding the port?

Reply:
before was hardcoded. now it's in settings and can be changed

Reply:
LGTM.